DigitalLearningSolutions.Web.Tests/TestHelpers/MyAccountHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.TestHelpers
+{
+    using DigitalLearningSolutions.Data.Models.User;
+
+    internal class MyAccountHelper
+    {
+        internal static DelegateUser GetDefaultDelegateUser
```

Comment:
Are these GetDefault...User not already present in the UserTestHelper in the data.tests project, or is there a benefit to having them in both test projects?